### ISSUE TITLE
timespan overflowed？

### ISSUE THREAD

**Issue by hualazimo7** [ClusterM/coolgirl-famicom-multicart#19]
I used S29gl01G, error happened when writing at sector 500-600.![mmexport1734924863060.png](https://github.com/user-attachments/assets/5ebbf835-815c-4845-983e-b4d8a00c8d08)
I have tested two chips on different boards, error still exsited

**Comment by ClusterM** [comment]
Weird error. Which client/dumper versions are you using? Are you really writing cartridge for **three hours**?

**Comment by hualazimo7** [comment]
> Weird error. Which client/dumper versions are you using? Are you really writing cartridge for **three hours**?

yeah, I have built a latest stm32&cpld dumper, while it is at my another house so I just use the slower ATMEGA dumper to write a 128MB rom.
The error happens.
Maybe the client has a limit for running duration？
Oh, the client is the latest version.

**Comment by ClusterM** [comment]
Did you tried again? Looks like a bad flash memory. You can specify a bad sector number. 
S29GL01G is not recommended at all.

**Comment by hualazimo7** [comment]
I got my stm32+cpld dumper yesterday, everything is ok with the same memory flash S29GL010g. It seems like an issue with the client.

**Comment by hualazimo7** [comment]
> Did you tried again? Looks like a bad flash memory. You can specify a bad sector number. 
> S29GL01G is not recommended at all.

I got my stm32+cpld dumper yesterday, everything is ok with the same memory flash S29GL010g. It seems like an issue with the client.
